librosa/core/audio.py

```diff
@@ -23,6 +24,8 @@
            'autocorrelate', 'lpc', 'zero_crossings',
            'clicks', 'tone', 'chirp']
 
+logger = logging.getLogger(__name__)
```

Comment:
We don't use logging anywhere else in the library -- why do this instead of warnings?

Reply:
`warnings` is nice! Just went with `logging` by habit.